HdmiCec/HdmiCec.cpp

```diff
@@ -107,6 +107,7 @@ namespace WPEFramework
 
         void HdmiCec::Deinitialize(PluginHost::IShell* /* service */)
         {
+            CECDisable();
```

Comment:
do we need this?
Entire HdmiCEC functionality is already in LgiHdmiCEC, right?

If this is bug, pleas create separate commit for this, and upstream this change to rdkcentral.